scripts/bookmarklet.tmpl.js

```diff
@@ -0,0 +1,54 @@
+/* global jQuery, window, document */
+(function(window, document) {
+  var toast = document.createElement('div');
+  var b = document.getElementsByTagName('body')[0];
+  var version = null;
+
+  setStyles(toast)([
+    ['position' , 'fixed'],
+    ['top' , '20px'],
+    ['right' , '30%'],
+    ['padding' , '5px 10px'],
+    ['zIndex' , 1001],
+    ['fontSize' , '18px'],
+    ['color' , '#222'],
+    ['backgroundColor' , '#DFF2BF'],
+    ['text-align', 'center'],
+    ['border', 'solid 2px black'],
+    ['border-radius', '25px']
+  ]);
```

Comment:
It seems to me this would be better as an object rather than an array of pairs. Also, why are some of the property names in camel case (e.g. `fontSize`) while others are in kebab case (e.g. `text-align`)?